concrete/elements/panels/dashboard.php

```diff
@@ -1,66 +1,94 @@
-<?php defined('C5_EXECUTE') or die("Access Denied.");
+<?php defined('C5_EXECUTE') or die('Access Denied.');
+//use Concrete\Core\Support\Facade\Application;
+use Concrete\Controller\Panel\Dashboard as DashboardPanel;
+
+//this file is the main dashboard panel element which is generated every page load and anytime panel is reopened
+// this is the view:element dashboard
 $parents = Loader::helper('navigation')->getTrailToCollection($c);
-$pageIDs = array();
+$pageIDs = [];
 foreach ($parents as $pc) {
     $pageIDs[] = $pc->getCollectionID();
 }
+$dashboard = new DashboardPanel();
+$checkForMenu = $dashboard->checkForMenu($c);
+if ($checkForMenu == 1) {
+    if ($tab == 'favorites') {
+        $wrapperClass = 'ccm-panel-content-visible';
+    } else {
+        $wrapperClass = 'ccm-panel-slide-left';
+    }
+} else {
+    $wrapperClass = 'ccm-panel-content-visible';
+}
```

Comment:
This whole if if else else comes down to 2 classes, so you can make it a one-line if/else:

`$wrapperClass = $checkForMenu === 1 && $tab !== 'favorites'  ?'ccm-panel-slide-left' : 'ccm-panel-content-visible';`

Also use triple equal sign for strict checking instead of double equal sign (more strict).

Reply:
Hey ramon, your code works as long as I keep it only double equal signs.  The actual checkForMenu function suggestions my mlocati work.  If I use your way then triple signs do work.